File: solutions/0001-two-sum/solution.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> solution;

        int len = nums.size();

        for (int i=0; i<len; i++) {
            for (int j=0; j<len; j++) {
                if (nums.at(i)+nums.at(j) == target) {
                    if (i != j) {
                        solution.push_back(i);
                        solution.push_back(j);

                        return solution;
                    }
                }
            }
        }

        return solution;
    }
};

File: solutions/0001-two-sum/solution.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        hashed = set(nums)
        for num in hashed:
            if (target - num) in hashed:
                first = nums.index(num)
                nums[first] = None
                if (target - num) in nums:
                    return [first, nums.index(target - num)]
